packages/maskbook/src/utils/native-rpc/types.ts

```diff
@@ -0,0 +1,27 @@
+import type * as settings from '../../settings/settings'
+/**
+ * JSON RPC calls that can be called from the native side.
+ * JS = server, native = client
+ */
+export interface WebviewAPIs {
+    web_echo<T>(arg: T): Promise<T>
+    getDashboardUrl(path: string): Promise<string>
```

Comment:
Please explicitly mark available "intent" instead of receiving a "path". BTW it should be named as `getDashboardURL`.

The API should looks like:

```ts
getDashboardURL(intent: "wallets" | "personas" | "other pages..."): Promise<string>
```

Reply:
@zhouhanseng For now please just remove the parameter, native only needs the dashboard root url

Reply:
ok

Reply:
done